python/pyspark/worker.py

```diff
@@ -177,8 +177,13 @@ def process():
             process()
     except Exception:
         try:
+            exc_info = traceback.format_exc()
+            try:
+                exc_info = exc_info.encode("utf-8")
```

Comment:
Benefit a lot, thx.

exc_info = exc_info.str(desc)    is right? I think I can remove it.

```python
if sys.version >= '3':
    unicode = str
if isinstance(exc_info, unicode):
    exc_info = exc_info.encode('utf-8'))

```